browser/themes/brave_theme_service_browsertest.cc

```diff
@@ -188,13 +188,21 @@ IN_PROC_BROWSER_TEST_F(BraveThemeServiceTest, OmniboxColorTest) {
       tp->GetColor(ThemeProperties::COLOR_OMNIBOX_RESULTS_BG));
 }
 
+// Some tests are failing for Windows x86 CI,
+// See https://github.com/brave/brave-browser/issues/22767
+#if BUILDFLAG(IS_WIN) && defined(ARCH_CPU_X86)
+#define MAYBE_DarkModeChangeByRegTest DISABLED_DarkModeChangeByRegTest
+#else
+#define MAYBE_DarkModeChangeByRegTest DarkModeChangeByRegTest
+#endif
+
```

Comment:
nit: usually there is no empty line between MAYBE_ defines and the test itself.

Reply:
thanks, I changed this also. In anyway I wanted to restart CI because of some macOS strange test failure